resources/lang/en/applicant/job_post.php

```diff
@@ -64,6 +64,22 @@
         'requirement_label' => 'Level Required: ',
         'level_link_title' => 'Visit the FAQ to learn about this and other skill levels.'
     ],
+    'language' => [
+        'sidebar_title' => 'View this job\'s language requirements.',
+        'title' => 'Language Requirements',
+        'english_icon_title' => 'An icon representing an English language requirement.',
+        'french_icon_title' => 'An icon representing an French language requirement.',
+        'english_essential_context' => 'This position has an English essential requirement. The manager may choose to test your proficiency in English.',
+        'french_essential_context' => 'This position has an French essential requirement. The manager may choose to test your proficiency in French.',
+        'bilingual_context_01' => 'This position has a bilingual requirement. This means that you must meet the %proficiency% required, in both English and French, in each of the 3 language skills: comprehension, writing, and oral communication.',
+        'proficiency_link' => 'https://www.canada.ca/en/public-service-commission/jobs/services/gc-jobs/information-candidates/language-requirements-candidates.html',
+        'proficiency_copy' => 'proficiency levels',
+        'bilingual_context_02' => 'You can try out a %assessment% to give yourself an idea of the level (A, B, or C) you might attain.',
```

Comment:
Just curious why this is wrapped in `%`. If it's meant to read "You can try out a assessment", it should be "You can try out **an** assessment".

Reply:
I agree it should read "an". Josh says the "%" add an anchor tag to the text.

Reply:
`english_french` isn't a viable option for Talent Cloud jobs. I believe it is a designation for pools of candidates who could be potentially placed in English or French environments based on their ability.  Gray has opened an issue to remove it from the app. https://github.com/GCTC-NTGC/TalentCloud/pull/752#issuecomment-470224769